columns/Column.js

```diff
@@ -121,6 +124,7 @@ export class Column {
         this.chooserGroup = chooserGroup;
         this.chooserDescription = chooserDescription;
         this.excludeFromChooser = withDefaultFalse(excludeFromChooser, false);
```

Comment:
realize that you did not add this, but line 126 looks like it has an extra argument